job_templates/packaging_job.xml.em

```diff
@@ -199,7 +199,11 @@ echo "# END SECTION"
 
 @[  if os_name in ['linux', 'linux-aarch64', 'linux-armhf', 'linux-centos']]@
 @[    if os_name in ['linux', 'linux-aarch64', 'linux-armhf']]@
+@[      if os_name == 'linux-armhf']@
```

Comment:
Might be cleaner not to nest this - `if os_name == linux-armhf; stuff; else if os_name in [linux, linux-aarch64]; other stuff`

Reply:
This was actually changed away from that approach to avoid duplicating the `DOCKER_BUILD_ARGS` in [the previous PR](https://github.com/ros2/ci/pull/299/files/6bd90f29c02ece30b0e0330eaa0bac295f0bd098#diff-526f24c7be4a93f0e5699f48a9f1656f). I'm inclined to keep the same approach here as was already committed there.

Reply:
:+1: I misread the nesting